mono/mini/method-to-ir.c

```diff
@@ -9281,22 +9332,28 @@ mono_method_to_ir (MonoCompile *cfg, MonoMethod *method, MonoBasicBlock *start_b
 					if (fsig->generic_param_count) {
 						/* virtual generic call */
 						g_assert (!imt_arg);
+						g_assert (will_have_imt_arg);
 						/* Same as the virtual generic case above */
 						imt_arg = emit_get_rgctx_method (cfg, context_used,
 														 cmethod, MONO_RGCTX_INFO_METHOD);
+						g_assert (imt_arg);
```

Comment:
Can this call ever return null ?

Reply:
Probably not. I added this implicit “pointer is/was boolean” notion so more sensitive to it.